seerep-srv/seerep-server/src/fb-meta-operations.cpp

```diff
@@ -144,4 +144,130 @@ grpc::Status FbMetaOperations::DeleteProject(grpc::ServerContext* context,
   return grpc::Status::OK;
 }
 
+grpc::Status
+FbMetaOperations::GetOverallTimeInterval(grpc::ServerContext* context,
+                                         const flatbuffers::grpc::Message<seerep::fb::UuidDatatypePair>* request,
+                                         flatbuffers::grpc::Message<seerep::fb::TimeInterval>* response)
+{
+  (void)context;  // ignore that variable without causing warnings
+  auto requestRoot = request->GetRoot();
+
+  std::string uuid = request->GetRoot()->projectuuid()->str();
+  boost::uuids::string_generator gen;
+  auto uuidFromString = gen(uuid);
+
+  std::vector<seerep_core_msgs::Datatype> dt_vector;
```

Comment:
please add a comment why you are using a vector, but only adds one element. Short summary of the problem with the fb enum